kolibri/plugins/coach/assets/src/modules/questionDetail/actions.js

```diff
@@ -19,7 +20,7 @@ export function setLearners(store, { questionId, exerciseId, quizId, classId, gr
   return resource
     .fetchCollection({
       getParams,
-      force: true,
+      force: !learnerId,
```

Comment:
same Q as before - would this break direct links to non-root pages?

Reply:
Nope!